R/replace_identified_by_x.R

```diff
@@ -0,0 +1,34 @@
+#' Replace each identified positions in protein by "x"
+#'
+#' Helper function for the calculation of sequence coverage, replaces identified positions with an "x"
```

Comment:
Helper function for the calculation of sequence coverage, replaces identified positions with an "x" within the protein sequence.